homeassistant/components/homekit_controller/button.py

```diff
@@ -151,6 +154,29 @@ async def async_press(self) -> None:
             await self.async_put_characteristics({key: val})
 
 
+class HomeKitProvisionPreferredThreadCrentials(CharacteristicEntity, ButtonEntity):
+    """Representation of a Button control for Ecobee clear hold request."""
```

Comment:
Think this comment is a copy/paste error

Reply:
It is indeed. The UI for this is still up in the air, if everything goes to plan you'll actually never see this button in a stable HA release.